src/lib/util/wizard-step.interface.spec.ts

```diff
@@ -51,37 +52,64 @@ class WizardTestComponent {
   }
 }
 
+@Component({
+  selector: 'aw-test-wizard',
+  template: `
+    <aw-wizard>
+      <aw-wizard-step stepTitle='Steptitle 1'>Step 1</aw-wizard-step>
+      <aw-wizard-step stepTitle='Steptitle 2' [completed]="true">Step 2</aw-wizard-step>
+      <aw-wizard-step stepTitle='Steptitle 3'>Step 3</aw-wizard-step>
+    </aw-wizard>
+  `
+})
+class WizardWithCompletedStepsTestComponent {
+  @ViewChild(WizardComponent)
+  public wizard: WizardComponent;
+}
+
 describe('WizardStep', () => {
-  let wizardTestFixture: ComponentFixture<WizardTestComponent>;
+  let wizardTestFixture: ComponentFixture<WizardTestComponent|WizardWithCompletedStepsTestComponent>;
 
   let wizardTest: WizardTestComponent;
   let wizard: WizardComponent;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [WizardTestComponent],
+      declarations: [WizardTestComponent, WizardWithCompletedStepsTestComponent],
       imports: [ArchwizardModule]
     }).compileComponents();
   }));
 
-  beforeEach(() => {
-    wizardTestFixture = TestBed.createComponent(WizardTestComponent);
-    wizardTestFixture.detectChanges();
+  const createTestWizard = () => {
+    const fixture = TestBed.createComponent(WizardTestComponent);
+    fixture.detectChanges();
 
-    wizardTest = wizardTestFixture.componentInstance;
+    wizardTestFixture = fixture;
+    wizardTest = fixture.componentInstance;
     wizard = wizardTest.wizard;
-  });
+  };
+
+  const createTestWizardWithCompletedSteps = () => {
+    const fixture = TestBed.createComponent(WizardWithCompletedStepsTestComponent);
+    fixture.detectChanges();
+
+    wizardTestFixture = fixture;
+    wizard = fixture.componentInstance.wizard;
+  };
 
   it('should create an instance', () => {
+    createTestWizard();
```

Comment:
Some of tests require another instance of wizard and call `createTestWizardWithCompletedSteps` instead (see the bottom of the file).

I think we should move those tests to a separate file. It will be uniform with navigation tests where we have separate `wizard-navigation.spec.ts`, `wizard-navigation-with-optional-step.spec.ts` etc. It will allow to move the `createTestWizard` / `createTestWizardWithCompletedSteps` calls to `beforeEach`. Do you agree?

Reply:
It seems like I overlooked the `createTestWizardWithCompletedSteps` method^^

Yes in this case I think it is the best approach to split the tests in two files.

Reply:
Done in 04256ba